### PR TITLE
feat: tech writer improvements to documentation (markdown) files

### PR DESCRIPTION
**Pull request by stephen-younge**
Issue #, if available: None

**Description of changes:**
Various changes to markdown files in `docs` directory to improve clarity and readability.

Built and deployed to docusauraus site at:
http://swb-documentation.s3-website-us-east-1.amazonaws.com/ 

Checklist: 

* [ Y ] Have you successfully deployed to an AWS account with your changes? 
* [ Y ] Have you linted your code locally prior to submission?
* [ N/A ] Have you written new tests for your core changes, as applicable?
* [ N/A ] Have you successfully ran unit tests and manual tests with your changes locally?
* [ N/A ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/sidebars.js [hunk 1]
```diff
@@ -16,9 +19,9 @@ module.exports = {
       {
         "Post Deployment": [
           "deployment/post_deployment/index",
+          "deployment/post_deployment/account_structure",
```
Comment (Bingjiling) [inline docs/sidebars.js:22]: File is not included with the PR?

File: docs/sidebars.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
-    ServiceWorkBench: ["introduction"],
+    ServiceWorkBench: ["introduction"],    
     "Deployment Guide": [
+      "deployment/about_this_guide",
       "deployment/summary",
+      "deployment/deployment_stages",      
       {
         "Pre Deployment": [
-          "deployment/pre_deployment/accounts",
+          "deployment/pre_deployment/predeployment",
```
Comment (Bingjiling) [inline docs/sidebars.js:10]: "deployment/pre_deployment/configuration"?

**Comment by stephen-younge** [comment]
@Bingjiling, I have addressed the issues with this PR, please take another look.

**Review by Bingjiling — APPROVED** [review]
Ship it!

**Pull request merged**